.gitlab/templates/pipeline.yaml.tpl

```diff
@@ -227,6 +227,34 @@ publish layer sandbox [us-east-1] ({{ $flavor.name }}):
   script:
     - .gitlab/scripts/publish_layers.sh
 
+publish layer sandbox [us-west-2] ({{ $flavor.name }}):
```

Comment:
I think this is unnecessary, since we already have a self monitoring stage, we can just reuse the same job and make it a range